contrib/ohi-release-notes/run.sh

```diff
@@ -119,7 +129,7 @@ fi
 
 
     # generating the changelog
-    ${RT_BIN} generate-yaml "$EXCLUDED_DIRECTORIES_FLAG" "$EXCLUDED_FILES_FLAG" "$INCLUDED_DIRECTORIES_FLAG" "$INCLUDED_FILES_FLAG"
+    ${RT_BIN} generate-yaml "$EXCLUDED_DIRECTORIES_FLAG" "$EXCLUDED_FILES_FLAG" "$INCLUDED_DIRECTORIES_FLAG" "$INCLUDED_FILES_FLAG" "$EXCLUDED_DEPENDENCIES_MANIFEST"
```

Comment:
This is not working as expected. That is the reason I gave @paologallinaharbur the tip of calling variables `$WHATEVER_FLAG`.

~~you are using the manifest file variable here, not the flag one. It is working because you are testing the defaults without passing a flag so at line 95 you are adding `--excluded-dependencies-manifest=` but in case a user sets its own file it stops working.~~ As 

This is hard to read, please, use a different variable for the file and the flag instead of reusing the same variable.